recipe/meta.yaml

```diff
@@ -1,69 +1,106 @@
 
 source:
   url: https://github.com/ibis-project/ibis/archive/{{ version }}.tar.gz
-  sha256: 36f99ad2a1dcd655d4e02557de01dd81055ea83fb9f3ed26954e4aec6b23ba78
+  sha256: e55178269266c8cd29b68bb5699fdeb251bcf9fdd4957e70a1f9916fd8ab4c18
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
-  skip: true  # [py<37]
+  number: 1
 
-requirements:
-  host:
-    - pip
-    - python
-    - setuptools
+outputs:
 
-  run:
-    - cached_property
-    - clickhouse-driver >=0.1.3
-    - clickhouse-cityhash  # [not win]
-    - clickhouse-sqlalchemy
-    - geoalchemy2
-    - geopandas
-    - graphviz
-    - impyla >=0.15.0
-    - lz4
-    - multipledispatch >=0.6
-    - numpy >=1.15
-    - pandas >=0.25.3
-    - parsy
-    - psycopg2
-    - pyarrow >=0.15
-    - pymysql
-    - pyspark >=2.4.3
-    - pytables >=3.0.0
-    - python
-    - python-graphviz
-    - python-hdfs >=2.0.16
-    - pytz
-    - regex
-    - requests
-    - shapely
-    - setuptools
-    - sqlalchemy >=1.1
-    - thrift >=0.11
-    - thriftpy2
-    - toolz
+  - name: ibis-framework
+    version: {{ version }}
 
-test:
-  imports:
-    - ibis
-    - ibis.backends.clickhouse
-    - ibis.backends.csv
-    - ibis.backends.parquet
-    - ibis.backends.hdf5
-    - ibis.backends.impala
-    - ibis.backends.mysql
-    - ibis.backends.pandas
-    - ibis.backends.postgres
-    - ibis.backends.pyspark
-    - ibis.backends.sqlite
+    build:
+      script: python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+      skip: true  # [py<37]
+
+    requirements:
+      host:
+        - pip
+        - python
+        - setuptools
+    
+      run:
+        - cached_property
+        - clickhouse-driver >=0.1.3
+        - clickhouse-cityhash  # [not win]
+        - clickhouse-sqlalchemy
+        - geoalchemy2
+        - geopandas
+        - graphviz
+        - impyla >=0.15.0
+        - lz4
+        - multipledispatch >=0.6
+        - numpy >=1.15
+        - pandas >=0.25.3
+        - parsy
+        - psycopg2
+        - pyarrow >=0.15
+        - pymysql
+        - pyspark >=2.4.3
+        - pytables >=3.0.0
+        - python
+        - python-graphviz
+        - python-hdfs >=2.0.16
+        - pytz
+        - regex
+        - requests
+        - shapely
+        - setuptools
+        - sqlalchemy >=1.1
+        - thrift >=0.11
+        - thriftpy2
+        - toolz
+    
+    test:
+      imports:
+        - ibis
+        - ibis.backends.clickhouse
+        - ibis.backends.csv
+        - ibis.backends.parquet
+        - ibis.backends.hdf5
+        - ibis.backends.impala
+        - ibis.backends.mysql
+        - ibis.backends.pandas
+        - ibis.backends.postgres
+        - ibis.backends.pyspark
+        - ibis.backends.sqlite
+
+  - name: ibis-framework-core
+    version: {{ version }}
+
+    build:
+      script: python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+      skip: true  # [py<37]
```

Comment:
This can probably be `noarch: python` with a minimum python version of 3.7

Reply:
thanks for the feedback. I will update the recipe! thanks!